alchemist-web-renderer/src/commonMain/kotlin/it/unibo/alchemist/common/utility/Ruotes.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2010-2023, Danilo Pianini and contributors
+ * listed, for each module, in the respective subproject's build.gradle.kts file.
+ *
+ * This file is part of Alchemist, and is distributed under the terms of the
+ * GNU General Public License, with a linking exception,
+ * as described in the file LICENSE in the Alchemist distribution's top directory.
+ */
+
+package it.unibo.alchemist.common.utility
+
+/**
+ * Objects that store all the route path. Since this object is part of the common sourceSet, both Client and Serve can
+ * refer to the correct Route maintaining consistency and reducing the number of possible errors and bug.
+ */
+object Ruotes {
+    /**
+     * Base environment path, for all Environment related operations.
+     */
+    private const val environmentPath: String = "/environment"
+
+    /**
+     * Route to get an Environment in a serialized form that needs to be rendered by the client, as
+     * [it.unibo.alchemist.common.model.RenderMode.CLIENT] was requested.
+     */
+    const val environmentClientPath: String = "$environmentPath/client"
+
+    /**
+     * Route to get an Environment already renderer by the Server, as
+     * [it.unibo.alchemist.common.model.RenderMode.SERVER] was requested.
+     */
+    const val environmentServerPath: String = "$environmentPath/server"
+
+    /**
```

Comment:
Can you explain better what a "simulation related operation" is?

Reply:
Operations that acts directly on the Simulation interface, calling its methods. Is this grouping ok? Do I need to change it or I just document it better?